cookbook/maintenance-mode.rst

```diff
@@ -0,0 +1,63 @@
+Sulu Maintenance Mode
+=====================
+
+When you need to deploy a new version of your project on a production environment
+it is often necessary to disable your sulu-application and inform your customers
+about it.
+
+Sulu maintenance mode displays a simple error page which can be easily customized.
+
+Create Maintenance Mode
+-----------------------
+
+To create a maintenance page, you first need to create a maintenance.php file:
+
+.. code-block:: bash
+
+    $ cp app/maintenance.php.dist app/maintenance.php
+
+Then you need to set the env variable SULU_MAINTENANCE to true. 
+E.g.: in your .htaccess file (for apache)
+ 
+.. code-block:: php
+
+    SetEnv SULU_MAINTENANCE true
+
+Configure Maintenance Mode
+--------------------------
+
+IP-Addreses:
+~~~~~~~~~~~~
+
+You may like to access your application while maintenance mode is active. Then you need to set $allowedIPs:
```

Comment:
I would only write allowed IPs, the name of the variable can be seen in the code block.